Package.swift

```diff
@@ -26,26 +26,25 @@ let package = Package(
       name: "Clocks",
       dependencies: [
         .product(name: "XCTestDynamicOverlay", package: "xctest-dynamic-overlay")
-      ],
-      swiftSettings: [
-        .unsafeFlags([
-          "-Xfrontend", "-warn-concurrency",
-          "-Xfrontend", "-enable-actor-data-race-checks",
-        ])
```

Comment:
also removed these flags cause it prevents building in Xcode. not sure why TCA CI didn't pick this up earlier, but I guess we need to do a 0.1.1 release after all 😆